src/lib/y2network/widgets/wireless_expert.rb

```diff
@@ -144,7 +144,7 @@ def maximum
       end
 
       def init
-        self.value = @settings.ap_scanmode
+        self.value = @settings.ap_scanmode.to_i
```

Comment:
Well, it should be an integer already, by default it is initialized to 1, but in the InterfaceFile it is declared as a String instead of an Integer. So, I guess we should unify it there.